src/levanter/models/attention.py

```diff
@@ -358,14 +358,13 @@ def _te_flash_attention(
         raise NotImplementedError("Using bias with flash attention on GPU is not currently implemented.")
 
     attn_output = fused_attn(
-        q=q_,
-        k=k_,
-        v=v_,
+        qkv=(q_, k_, v_),
         bias=fused_attn_bias,
         mask=fused_attn_mask,
         seed=prng,
         attn_bias_type=attn_bias_type,
         attn_mask_type=attn_mask_type,
+        qkv_layout=QKVLayout.BSHD_BSHD_BSHD,
```

Comment:
do you happen to know what the fastest layout is on nvidia? maybe it doesn't matter so much

Reply:
I sure don't! If I end up benchmarking this I'll let you know.